ui/.env

```diff
@@ -0,0 +1,5 @@
+# Note: After changing this file, must restart React app. Reloading Chrome tab
```

Comment:
Will we need to restart the data explorers deployed on GCP?

Reply:
No, only need to restart if we want them to have a particular commit